modules/openapi-generator/src/main/resources/Java/libraries/jersey2/pojo.mustache

```diff
@@ -189,6 +189,7 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#vendorE
    * @return {{name}}
   **/
 {{#required}}
+  @javax.annotation.Nonnull
```

Comment:
Do we need to wrap it in `{{^isNullable}} ... {{/isNullable}}` instead? Current change will result in the following for non-nullable required property?

``` 
    @javax.annotation.Nonnull
    @javax.annotation.Nullable
```

Reply:
Thanks for the hint! 👍 

> Do we need to wrap it in `{{^isNullable}} ... {{/isNullable}}` instead? 

Not instead but in addition.

> Current change will result in the following for non-nullable required properties?

Didn't you mean "required nullable properties"?

Either way, I'll fix this flaw with the next commit.